Client/src/components/Shared/Navbar/Navbar.jsx

```
@@ -1,135 +1,158 @@
+              </div>
             ) : (
-              <button 
-                onClick={() => window.location.href = '/login'}
-                className="px-6 py-2.5 bg-blue-600 text-white text-sm font-medium rounded-full hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2 transition-all duration-200 ease-in-out"
+              <Link 
+                to="/login"
+                className="relative group px-6 py-2.5 bg-blue-600 rounded-full text-white text-sm font-medium shadow-md hover:shadow-lg transition-all duration-300"
               >
-                Login
-              </button>
+                <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-full shadow-md"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)